### PR TITLE
Broadcast and auth

### PR DESCRIPTION
**Pull request by mpuel**
Hi,

this patch enables two configuration options in ntp.conf : the broadcast mode (the simple way) and the disabling of authentication.

Would you mind envisaging to merge those changes ?

### CODE REVIEW

**Comment by igalic** [comment]
do you think you can add a test to [spec/classes/ntp_spec](https://github.com/puppetlabs/puppetlabs-ntp/blob/master/spec/classes/ntp_spec.rb)?

**Comment by mpuel** [comment]
Ok; I added 4 tests based on the value of the two parameters. It appears to apply fine but I am unsure about the placement in the ntp_spec file, please review it with cautious eyes :)

File: templates/ntp.conf.erb [hunk 1]
```diff
@@ -27,6 +30,10 @@ interface listen <%= interface %>
 <% end -%>
 <% end -%>
 
+<% if @broadcastclient == true %>
```
Comment (igalic) [inline templates/ntp.conf.erb]: please turn this `%>` into a `-%>`

File: templates/ntp.conf.erb [hunk 2]
```diff
@@ -27,6 +30,10 @@ interface listen <%= interface %>
...
+broadcastclient
+<% end %>
```
Comment (igalic) [inline templates/ntp.conf.erb]: please turn this `%>` into a `-%>`

**Comment by igalic** [comment]
looks good to me (except for the comment above).
please [squash](http://www.git-scm.com/book/en/Git-Tools-Rewriting-History#Squashing-Commits) your commits again down to ~one.

**Comment by mpuel** [comment]
Ok, done.

**Pull request merged**